Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml

```
@@ -291,8 +291,7 @@
   id: HeadRevGear
   storage:
     back:
-    - Flash
-    - ClothingEyesGlassesSunglasses
+    - RevolutionaryManifesto
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)